server/db.js

```diff
@@ -0,0 +1,15 @@
+const { Pool } = require('pg');
+
+link = "postgres://vtsohdxm:jNSKq4-96QSdfZax4qdrprDhVwu-rcrF@surus.db.elephantsql.com/vtsohdxm"
```

Comment:
You probably don't want to check this in, containing a password - can you use the `dotenv` library and a `.env` file to set this up instead?